components/Movie.js

```diff
@@ -29,13 +29,34 @@ const Id = styled.p`
 `;
 
 const Movie = ({ title, description, releaseYear, id }) => {
```

Comment:
I think the fishes prefer:

export default function Movie({ title, description, releaseYear, id }) {
return(
...Your code...
);}